src/overlays/ovl_i2/fox_me.c

```diff
@@ -1,97 +1,2527 @@
+            Math_SmoothStepToF(&D_80177A48[2], 1500.0f, 0.1f, 10.0f, 0.0f);
+            Math_SmoothStepToF(&D_80177A48[0], 0.1f, 0.1f, 0.001f, 0.0f);
+            Math_SmoothStepToF(&player->unk_0D0, 0.0f, 1.0f, 0.5f, 0.0f);
+            Math_SmoothStepToF(&gBosses->vel.z, 0.0f, 1.0f, 0.5f, 0.0f);
+            Math_SmoothStepToF(&player->unk_0E4, 0.0f, 0.1f, 3.0f, 0.0f);
+
+            if (D_80177A80 == 320) {
+                D_80177A80 = 200;
+                D_80177A48[0] = 0.0f;
+                D_80177A48[3] = 0.0f;
+                D_80177A48[4] = 0.0f;
+                player->unk_1D0++;
+                player->timer_1F8 = 100;
+            }
+            break;
+
+        case 2:
+            if (player->timer_1F8 == 0) {
+                Math_SmoothStepToF(&D_80177A48[3], 1.0f, 1.0f, 0.01f, 0.0f);
+            }
+            if (player->timer_1F8 == 50) {
+                func_8001D444(0, 0x26U, 0, 0xFFU);
+            }
+
+            Math_SmoothStepToF(&D_80177A48[1], 1080.0f, 0.01f, D_80177A48[3], 0.0f);
+
+            if (D_80177A80 < 500) {
+                Math_SmoothStepToF(&D_80177A48[2], -1700.0f, 0.1f, 30.0f, 0.0f);
+            } else if (D_80177A80 < 800) {
+                Math_SmoothStepToF(&D_80177A48[2], -1000.0f, 0.1f, 4.0f, 0.0f);
+            } else if (D_80177A80 < 1150) {
+                Math_SmoothStepToF(&D_80177A48[2], -1250.0f, 0.1f, 4.0f, 0.0f);
+            } else {
+                Math_SmoothStepToF(&D_80177A48[2], -1500.0f, 0.1f, 4.0f, 0.0f);
+            }
+
+            if (D_80177A80 < 400) {
+                Math_SmoothStepToF(&D_80177A48[4], 0.0f, 0.03f, 0.2f, 0.0f);
+            } else if (D_80177A80 < 700) {
+                Math_SmoothStepToF(&D_80177A48[4], -40.0f, 0.03f, 0.4f, 0.0f);
+            } else if (D_80177A80 < 1100) {
+                Math_SmoothStepToF(&D_80177A48[4], 40.0f, 0.03f, 0.4f, 0.0f);
+            } else {
+                Math_SmoothStepToF(&D_80177A48[4], 7.0f, 0.03f, 0.2f, 0.0f);
+            }
+
+            Math_SmoothStepToF(D_80177A48, 0.1f, 0.1f, 0.002f, 0.0f);
+
+            Matrix_RotateX(gCalcMatrix, D_80177A48[4] * M_DTOR, 0);
+            Matrix_RotateY(gCalcMatrix, D_80177A48[1] * M_DTOR, 1);
+
+            src.x = 0.0f;
+            src.y = 0.0f;
+            src.z = D_80177A48[2] + 2000.0f;
+
+            Matrix_MultVec3f(gCalcMatrix, &src, &dest);
+
+            D_801779A0 = player->pos.x;
+            D_801779B8 = player->pos.y;
+            D_801779C0 = player->unk_138 + D_80177D20 + 150.0f;
+
+            if (D_80177A80 > 1390) {
+                player->unk_0D0 += 2.0f;
+                player->unk_0E4 += 0.1f;
+                player->unk_190 = 2.0f;
+
+                if (D_80177A80 == 1465) {
+                    func_8001DBD0(20);
+                }
+
+                if (D_80177A80 >= 1466) {
+                    D_80178358 = 0xFF;
+                    D_80178348 = D_80178350 = D_80178354 = 0;
```

Comment:
```suggestion
                    D_80178358 = 255;
                    D_80178348 = D_80178350 = D_80178354 = 0;
```
these guys are all color values